### ISSUE TITLE
Can I use this code to generate a simulated GNSS/GPS signal?

### ISSUE THREAD

**Issue by Imtiaz08** [danipascual/GNSS-matlab#3]
I have already constructed a GPS signal but when I read it through GNSS SDR it says the preamble is invalid which means my subframe data needs reworking. 

Can you please tell me if it is possible to use this code along with RINEX files to generate real satellite signal?

**Comment by sticky-tea** [comment]
Hey! Have you found a solution?

**Comment by Imtiaz08** [comment]
yes I was able to spoof multiple devices during my masters thesis

**Comment by sticky-tea** [comment]
> yes I was able to spoof multiple devices during my masters thesis

Can I read your masters thesis?

**Comment by FayeWong98** [comment]
@Imtiaz08 @sticky-tea 
Hello, have you solved this problem? Using generated signals to spoof devices.

**Comment by Imtiaz08** [comment]
Yes I did it for my MS thesis

**Comment by FayeWong98** [comment]
Nice to receive your reply! This is also a problem that has been bothering me all along. I am a beginner in this field. Can you provide some help or provide an email for consultation.

Wishing you a happy and fulfilling life!

**Comment by Imtiaz08** [comment]
Yes sure you can email me at imtiaznabi8@gmail.com
